lib/cli/interactive-setup/deploy.js

```diff
@@ -0,0 +1,188 @@
+    process.stdout.write(`\n  Run ${chalk.bold('serverless deploy')} in the directory\n`);
+    process.stdout.write(
+      "    Redeploy your service after you've updated your service code or configuration\n"
+    );
+    process.stdout.write(
+      `\n  Run ${chalk.bold('serverless invoke')} and ${chalk.bold(
+        'serverless logs'
+      )} in the project directory\n`
+    );
+    process.stdout.write('    Invoke your functions directly and view the logs\n');
+    process.stdout.write(`\n  Run ${chalk.bold('serverless')} in the project directory\n`);
+    process.stdout.write(
+      '    Add metrics, alerts, and a log explorer, by enabling the dashboard functionality\n\n'
+    );
+    return;
+  }
+
+  process.stdout.write(
+    `\n${chalk.green('Your project is ready for deployment and available in ')}${chalk.white.bold(
+      `./${serviceName}`
+    )}\n`
+  );
+  process.stdout.write(`\n  Run ${chalk.bold('serverless deploy')} in the project directory\n`);
+  process.stdout.write('    Deploy your newly created service\n');
+  process.stdout.write(
+    `\n  Run ${chalk.bold('serverless info')} in the project directory after deployment\n`
+  );
+  process.stdout.write('    View your endpoints and services\n');
+  process.stdout.write(
+    `\n  Run ${chalk.bold('serverless invoke')} and ${chalk.bold(
+      'serverless logs'
+    )} in the project directory after deployment\n`
+  );
+  process.stdout.write('    Invoke your functions directly and view the logs\n');
+  process.stdout.write(`\n  Run ${chalk.bold('serverless')} in the project directory\n`);
+  process.stdout.write(
+    '    Add metrics, alerts, and a log explorer, by enabling the dashboard functionality\n\n'
+  );
+};
+
+const configurePlugin = (serverless, originalStdWrite) => {
+  serverless.pluginManager.addPlugin(require('../../plugins/deploy-interactive'));
+  const interactivePlugin = serverless.pluginManager.plugins.find(
+    (plugin) => plugin.constructor.name === 'InteractiveDeployProgress'
+  );
+  interactivePlugin.progress._writeOriginalStdout = (data) => originalStdWrite(data);
+  return interactivePlugin;
+};
+
+module.exports = {
+  async isApplicable({ configuration, serviceDir, history, options }) {
+    if (!serviceDir) {
+      return false;
+    }
+
+    // We only want to consider newly created services for deploy step
+    if (!history.has('service')) {
+      return false;
+    }
+
+    if (
+      _.get(configuration, 'provider') !== 'aws' &&
+      _.get(configuration, 'provider.name') !== 'aws'
+    ) {
+      return false;
+    }
+
+    // If `awsCredentials` step was not executed, we should proceed as it means that user has available credentials
+    if (!history.has('awsCredentials')) return true;
+
+    // We want to proceed if the service instance has a linked provider
+    if (
+      isUserLoggedIn() &&
+      (await doesServiceInstanceHaveLinkedProvider({ configuration, options }))
```

Comment:
Is it safe to run when there's no `configuration.org` (?)

Reply:
No, we should first check if there's `org` configured, I'm going to change it :+1: